### PR TITLE
ultra range again

### PR DESCRIPTION
**Pull request by xxalpha**
New ``urange()`` proc based off ``block()``. Faster than previous ``ultra_range()`` (now ``spiral_range()``). Like the previous proc, it's faster for ranges of 9 and higher, tied at 8, slower for 7 and lower.

Supports including areas at the cost of performance, but still faster than ``range()`` and not limited to 30 range.

This will compile when #15162 is merged.

### CODE REVIEW

**Comment by duncathan** [comment]
What's the purpose of keeping the old one if it's not even used?

**Comment by xxalpha** [comment]
In case you want a list organized by closeness. I still plan on using it, I temporarily stopped working on the thing I originally made the proc for.

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -1117,8 +1117,27 @@ B --><-- A
...
+			return list()
+
+	var/list/turfs = RANGE_TURFS(dist, center)
+	if(orange)
+		turfs -= get_turf(center)
+	. = list()
+	for(var/V in turfs)
+		var/turf/T = V
+		. += T
+		. += T.contents
```
Comment (tkdrg) [inline code/__HELPERS/unsorted.dm]: what about contents of contents, etc

**Comment by tkdrg** [comment]
Also conflicts

File: code/__HELPERS/unsorted.dm (hunk 1 above)
Comment (RemieRichards) [inline code/__HELPERS/unsorted.dm]: @tkdrg range doesn't return those, does it?

Comment (xxalpha) [inline code/__HELPERS/unsorted.dm]: No it doesn't.

**Comment by duncathan** [comment]
code/datums/diseases/wizarditis.dm:91:error: ultra_range: undefined proc
code/game/mecha/equipment/tools/other_tools.dm:40:error: ultra_range: undefined proc

@xxalpha

**Pull request merged**